MAINTAINERS.yml

```diff
@@ -490,6 +490,7 @@ Documentation:
     - doc/releases/
     - doc/security/
     - README.rst
+    - .clang-format
```

Comment:
I do not think `.clang-format` belongs under the "Documentation" area.

Reply:
My thinking is that if we make `.clang-format` the source of truth, then it we want the same maintainers in the loop for `.clang-format` changes as this could require a documentation update to match.